packages/plugin-divider/src/DividerXmlDomParser.ts

```diff
@@ -14,6 +15,11 @@ export class DividerXmlDomParser extends AbstractParser<Node> {
     async parse(item: Element): Promise<DividerNode[]> {
         const divider = new DividerNode();
         divider.modifiers.append(this.engine.parseAttributes(item));
+
+        // `DividerNode` can be configured to be unbreakable.
+        const configuration: DividerConfig = this.engine.editor.plugins.get(Divider).configuration;
+        divider.breakable = 'breakable' in configuration ? !!configuration.breakable : true;
```

Comment:
I think you should do a `defaultConfig` like you did somewhere else and do something like:
`this.config = { ...this.defaultConfig, config }`
Such that we can simply do `divider.breakable = configuration.breakable` here.

Reply:
Done.